.github/workflows/test-translations.yml

```
@@ -58,16 +58,21 @@ jobs:
         ref: ${{ env.I18N_BRANCH }}
 
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v5
       with:
         python-version: >-
           3.10
 
     - name: Install Python tooling
       run: python -m pip install --upgrade nox virtualenv sphinx-lint
 
+    - name: Set Sphinx problem matcher
+      uses: sphinx-doc/github-problem-matcher@master
```

Maybe, pin to a commit hash?

Also, let's add this one to other workflows that run Sphinx, right in this PR.

---

prefer a commit or `v0` (for the current v0.1)?

---

I contributed the conversion to composite actions, which made it much faster and cross-compatible, and asked for a new tag. So it's now
```suggestion
      uses: sphinx-doc/github-problem-matcher@v1.0
```